app/src/main/java/org/apache/fineract/data/services/TellersService.kt

```diff
@@ -0,0 +1,18 @@
+package org.apache.fineract.data.services
+
+import io.reactivex.Observable
+import org.apache.fineract.data.models.teller.Teller
+import org.apache.fineract.data.remote.EndPoints
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface TellersService {
+
+    @GET(EndPoints.API_TELLER_PATH + "/offices" + "/{officeIdentifier}" + "/teller")
+    fun getTellerList(@Path("officeIdentifier") officeIdentifier: String): Observable<List<Teller>>
+
+    @GET(EndPoints.API_TELLER_PATH + "/offices" + "/{officeIdentifier}" +
```

Comment:
you don't need to split. `/offices/{officeIdentifier}/teller/{tellerCode}` is fine and standard.